planetary_test_data/planetary_test_data.py

```diff
@@ -1 +1,66 @@
 # -*- coding: utf-8 -*-
+import os
+import json
+import urllib
+import planetary_test_data
+import shutil
+
+"""
```

Comment:
Please include both of these comments as "docstrings" https://www.python.org/dev/peps/pep-0257/ in their functions.